virtual_rainforest/data_variables.toml

```diff
@@ -366,29 +405,37 @@ initialised_by = "abiotic"
 updated_by = "abiotic"
 used_by = [ "abiotic"]
 
+[[variable]]
+name = "vertical_flow"
+description = "Vertical flow of water through soil column"
+unit = "mm"
```

Comment:
If it's a flow should the unit be distance per time?

Reply:
Good point. At the moment, it is mm/month, but we could convert that in whatever is useful, like mean flow in m3/s